helper-commands.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Helper Tools and Commmands
+updated_at: 2023-10-15
+extra_styles: roles
+---
+
+This page details the tools that are available to helpers within help channels and the help forum.
+
+## Commands
+
+### `.close`
+
+An "occupied" or "pending" channel can be closed at any time by either its owner or by people with the <a class="mention helpful">@Helpful</a> role using the `.close` command. When used in [#help-forum](){: .mention}, this functions the same as `.solved`.
+
+### `.reopen`
+
+If a channel is "closed" or "available", it can be reopened by the previous owner or by people with the <a class="mention helpful">@Helpful</a> role using the `.reopen` command.  Note that, after a short period following closure, help channels are locked and are unable to be reopened. When used in [#help-forum](){: .mention}, this functions the same as `.unsolved`.
+
+### `.solved`
+
+Marks a forum post as solved. Can be used by the previous owner or by people with the <a class="mention helpful">@Helpful</a> role.
+
+### `.unsolved`
+
+Mark a forum post as unsolved. Can be done by the owner of the thread or by people with the <a class="mention helpful">@Helpful</a> role.
+
+## Factoids
+
+A factoid is a short command that can be used to prompt <a class="mention">@Bot</a> to recall a single message containing text. To use a factoid, simply send a message containing the factoid command and no other text. Below is a list of all help-related factoids, alongside their content. These can be used by all users. 
+
+### `!15m`
+> Please only use the Helpers ping once if your question has not been answered for 15 minutes. Please do not ping or DM individual users about your question.
```

Comment:
So then do we update this page every time we update factoids?

Reply:
That is the intention - helper factoids are added fairly infrequently so I think this is reasonable.